src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/ocr/OcrDataField.java

```diff
@@ -1,31 +1,32 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.model.ocr;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonSetter;
 import com.fasterxml.jackson.annotation.Nulls;
-import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.TextNode;
 import com.fasterxml.jackson.databind.node.ValueNode;
 
 public class OcrDataField {
 
-    private TextNode name;
-    private ValueNode value;
+    @JsonProperty(value = "metadata_field_name", required = true)
+    private TextNode metadataFieldName;
+    private ValueNode metadataFieldValue;
```

Comment:
we can keep the field names unchanged. IMHO it will be clearer this way.

Reply:
don we not need `@JsonProperty(value = "metadata_field_value")` on this one?

Reply:
No, we don't need `@JsonProperty` here. forgot to remove it, thanks!

Reply:
> we can keep the field names unchanged. IMHO it will be clearer this way.

Do you mean the field names should be `name` and `value`? 
But we want to send the ocr data to services without making any changes. isn't it?

Reply:
that's why I suggested adding `@JsonProperty(value = "metadata_field_value")` so that it gets serialized to JSON in original format.

Reply:
I tried it but it's not working